### PR TITLE
Update OmniChromium manifest for new Codesearch URL.

### PR DESCRIPTION
**Pull request by reillyeon**
Sorry, one more change needed to make this work. I assumed the CORS error I got after updating the URL was a server-side issue that I'd have to bug the Infra team about.

### CODE REVIEW

**Comment by kalman** [comment]
Ok could you change version to 12 in this?

**Comment by kalman** [comment]
you'll need to sync anyway

**Comment by reillyeon** [comment]
Done, and fixed the other URLs while I was at it. The only thing still broken is author search. Monorail returns an HTTP 400 when OmniChromium makes the request but 200 when I make it myself with wget.

File: omni-chromium/author.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function AuthorSearcher(query) {
 inherits(AuthorSearcher, Searcher);
 
 AuthorSearcher.prototype.getSuggestionsURL = function() {
-  return 'https://code.google.com/p/chromium/feeds/issueOptions';
+  return 'https://bugs.chromium.org/p/chromium/feeds/issueOptions';
```
Comment (kalman) [inline omni-chromium/author.js]: Yeah, pity this doesn't work. Would be nice to debug at some point, weird that exatly the same request works with wget.

**Pull request merged**